plugin/externalPluginRoot.go

```diff
@@ -99,27 +99,27 @@ func (r *externalPluginRoot) WrappedTypes() SchemaMap {
 
 // partitionSchemaGraph partitions graph into a map of <type_id> => <schema_graph>
 func (r *externalPluginRoot) partitionSchemaGraph(graph *linkedhashmap.Map) map[string]*linkedhashmap.Map {
-	var populate func(*linkedhashmap.Map, entrySchema, map[string]bool)
-	populate = func(g *linkedhashmap.Map, node entrySchema, visited map[string]bool) {
-		if visited[node.TypeID] {
+	var populate func(*linkedhashmap.Map, string, entrySchema, map[string]bool)
```

Comment:
Why pre-declare the variable when you create it on the next line?

Reply:
That's the Go workaround of getting recursion to work w/ closures.